crates/proto/src/serialize/txt/trust_anchor.rs

```diff
@@ -0,0 +1,285 @@
+//! Trust anchor file parser
```

Comment:
Is the trust anchor name from the specifications? I feel like it is a little confusing relative to TLS trust anchors (and specifically the `rustls_pki_types::TrustAnchor` type which we also use when setting up TLS connections).

Reply:
RFC8499 which is the best current practice RFC on DNS terminology defines 'trust anchor' as:

> A configured DNSKEY RR or DS RR hash of a DNSKEY RR. A validating security-aware resolver uses this public key or hash as a starting point for building the authentication chain to a signed DNS response.  In general, a validating resolver will have to obtain the initial values of its trust anchors via some secure or trusted means outside the DNS protocol.

so yes "trust anchor" in the context of DNSSEC refers to public key material in DNSKEY or DS records.

A few RFCs are bit more specific and say "DNSSEC trust anchor". Maybe we can use that more specific terminology here in the docs?

> specifically the rustls_pki_types::TrustAnchor type

there's also a `TrustAnchor` type in `proto::rr::dnssec::TrustAnchor`. it's in a `dnssec` module so that helps it disambiguate it from the TLS `TrustAnchor`.

that actually makes me think: it may be better to directly return a `TrustAnchor` from `parse`. that way we about introducing a new `Entry` type.

Reply:
> that actually makes me think: it may be better to directly return a TrustAnchor from parse. that way we about introducing a new Entry type.

decided to keep things simple as they currently are since that's easier to test